src/server/game/Entities/Player/Player.cpp

```diff
@@ -19538,6 +19538,36 @@ void Player::AddInstanceEnterTime(uint32 instanceId, time_t enterTime)
         _instanceResetTimes.insert(InstanceTimeMap::value_type(instanceId, enterTime + HOUR));
 }
 
+WorldSafeLocsEntry const* Player::GetInstanceEntrance(uint32 targetMapId)
+{
+    if (GetMapId() == targetMapId)
+        return nullptr;
```

Comment:
Seems reasonable to support fetching current entrance while inside the same map, any specific reason to why this should be prevented?

Reply:
agreed